src/LanguageServer/Protocol/Features/Diagnostics/EngineV2/DiagnosticIncrementalAnalyzer.HostAnalyzerInfo.cs

```diff
@@ -17,13 +17,14 @@ private partial class DiagnosticIncrementalAnalyzer
     {
         private partial class StateManager
         {
-            private HostAnalyzerInfo GetOrCreateHostAnalyzerInfo(Project project, ProjectAnalyzerInfo projectAnalyzerInfo)
+            private HostAnalyzerInfo GetOrCreateHostAnalyzerInfo(
+                SolutionState solution, ProjectState project, ProjectAnalyzerInfo projectAnalyzerInfo)
```

Comment:
note: this follows the pattern of using 'solution/project' as the name of hte 'State' instance instead of 'solutionState/projectState'.  We only use the latter if we have code that uses both at the same time.